Configs/.config/hyde/wallbash/Wall-Ways/code.dcol

```diff
@@ -1,4 +1,4 @@
-${cacheDir}/landing/wallbashcode.json|${scrDir}/wallbashcode.sh
+$HOME/.vscode-oss/extensions/undefined_publisher.wallbash-0.0.1/themes/wallbash-color-theme.json
```

Comment:
any reason to change this? it was change to `${cacheDir}/landing/wallbashcode.json` to support `code-oss` and `vscode-bin`

Reply:
that must have slipped in when i updated my branch will stash the change